catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlResult.java

```diff
@@ -63,6 +64,8 @@
     private static final Map<String, String> SPATIAL_CONTEXT_ARGUMENTS = ImmutableMap.of(
             "spatialContextFactory",
             JtsSpatialContextFactory.class.getName(),
+            "validationRule",
+            ValidationRule.repairConvexHull.name(),
```

Comment:
Do we need `MultiOverlap` here too?

We should probably note why we are performing `ConvexHull` here, and why we are not using `Buffer0`. There is some discussion on it here: https://github.com/locationtech/spatial4j/issues/34#issuecomment-12426283